common/src/app/common/spec/radius.cljc

```diff
@@ -30,6 +31,10 @@
 ;; shapes that has border radius, and so it hasn't :rx nor :r1. 
 ;; In this case operations must leave shape untouched.
 
+(defn has-radius?
+  [shape]
+  ((get editable-attrs (:type shape)) :rx))
```

Comment:
I think is better to use `contains?` here.

Reply:
done